README.md

```diff
@@ -265,17 +265,24 @@ To restore in different namespace, run the following command:
 velero restore create --from-backup backup_name --restore-volumes=true --namespace-mappings source_ns:destination_ns
 ```
 
-*Note: After restore for remote backup is completed, you need to set target-ip for the volume in pool pod. If restore is from local snapshot then you don't need to update target-ip*
-*Steps to get target-ip*
-1. kubectl get svc -n openebs <PV_NAME> -ojsonpath='{.spec.clusterIP}'
-*Steps to update `target-ip` in pool pod is as follow:*
+Once restore for remote backup is completed, You need to set targetip in relevant replica. Refer [Setting targetip in replica](#setting-targetip-in-replica).
```

Comment:
Make this bold. **Once restore for remote backup is completed, You need to set targetip in relevant replica. Refer [Setting targetip in replica](#setting-targetip-in-replica).**